### PR TITLE
Dala 331 linkedin login

### PR DESCRIPTION
**Pull request by MarcMDfine**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] ~At least one E2E Test exists testing the new feature~
- [x] Documentation is updated as required
- [x] ~The automated deployment is updated if required~
- [x] The new version is deployed to the dev server using this branch
  - [x] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [x] The new feature is manually used/tested/observed on dev server
  - [x] All implemented Social Logins have been tested manually in the UI
- [x] ~There is at least one picture for each story, which was created before coding has started~
- [x] If any work on the UI is to be merged, those changes were also documented in the Figma 
- [x] The local Dev stack still works: execute `startDevelopmentStack.sh`, npm serve, npm teste2e and execute Cypress Tests locally

### CODE REVIEW

File: dataland-keycloak/dataland_theme/login/templates/login.ftl [hunk 1]
```diff
@@ -77,15 +77,28 @@
...
+                        <div id="social-${p.alias}">
+                            <button
+                                    label="Join"
+                                    class="p-button cursor-pointer w-full bg-white text-primary uppercase mt-3"
+                                    onclick="location.href='${p.loginUrl}'"
+                            >
+                                LOG IN WITH ${p.displayName!}
+                            </button>
+                        </div>
+                    </#if>
```
Comment (emanueldfine) [inline dataland-keycloak/dataland_theme/login/templates/login.ftl:101]: Sicher, dass man das nicht weiterhin mit einem generischen Code-Block abfrühstücken kann?

Z.B. nur das image für den case LinkedIn conditionally rendern?

File: dataland-keycloak/dataland_theme/login/templates/register.ftl [hunk 2]
```diff
@@ -81,15 +81,28 @@
...
+                        <div id="social-${p.alias}">
+                            <button
+                                    label="Join"
+                                    class="p-button cursor-pointer w-full bg-white text-primary uppercase mt-3"
+                                    onclick="location.href='${p.loginUrl}'"
+                            >
+                                JOIN WITH ${p.displayName!}
+                            </button>
+                        </div>
+                    </#if>
```
Comment (emanueldfine) [inline dataland-keycloak/dataland_theme/login/templates/register.ftl:105]: Same wie oben

File: dataland-keycloak/dataland_theme/login/templates/login.ftl (hunk 1 above)
Comment (MarcMDfine) [reply dataland-keycloak/dataland_theme/login/templates/login.ftl:101]: Ja. Könnte man machen. Würde dann aber (meiner Meinung nach) schlechter Lesbar sein. Gerade weil die Knöpfe auch unterschiedliche Klassen haben und evtl. auch noch ein Icon für das Google-Login dazu kommen könnte.

Aber wenn du da anderer Meinung bist, kann ich das auch gerne eben ändern

File: dataland-keycloak/dataland_theme/login/templates/login.ftl [hunk 3]
```diff
@@ -77,15 +77,28 @@
...
-                    <div id="social-${p.alias}">
-                        <button
-                            label="Join"
-                            class="p-button cursor-pointer w-full bg-white text-primary uppercase"
-                            onclick="location.href='${p.loginUrl}'"
-                        >
-                            LOG IN WITH ${p.displayName!}
-                        </button>
-                    </div>
+                    <#if p.alias="linkedin">
```
Comment (emanueldfine) [inline dataland-keycloak/dataland_theme/login/templates/login.ftl:80]: Make LinkedIn picture appear in User Dropdown after login via LinkedIn

Comment (emanueldfine) [reply dataland-keycloak/dataland_theme/login/templates/login.ftl:80]: Postponed due to much more complicated implementation than for Google Social Login

**Pull request merged**